packages/forklift-console-plugin/src/modules/Providers/views/list/ProvidersListPage.tsx

```diff
@@ -179,11 +179,16 @@ const ProvidersListPage: React.FC<{
     namespace,
   });
 
+  //  A list of providers belong to the active namespace
+  const currNamespaceProviders: V1beta1Provider[] = providers.filter(
+    (provider) => provider.metadata?.namespace === namespace,
```

Comment:
this should not work when namespace is "all" , because each provider will have it's own namespace, for example "openshfit-mtv" and in the case of "all" we will not get this provider, but but we do want it.

so in the case we want to show only one namespace, we already have only the providers of this namespace, and we don't need this filtering.
and in the case of "all" this filter also does not add any functionality, because it will give an empty list.

I see that in `useProvidersInventoryLis` you will ignore this list in the case of "all", so we don't actually need to filter the providers in the case of "all" , and we don't need to filter them in the case of specific namespace. 

in what case do we need to filter providers by namespace ?